openwisp_radius/urls.py

```diff
@@ -13,31 +12,18 @@ def get_urls(api_views=None, social_views=None, saml_views=None):
     Arguements:
         api_views(optional): views for Radius API
         social_view(optional): views for social login (if enabled)
+        saml_views(optional): views for saml login (if enabled)
     """
-    if not social_views:
-        from .social import views as social_views
-    urls = [
+    return [
         path('api/v1/', include(get_api_urls(api_views))),
-        path(
-            urljoin(app_settings.CSV_URL_PATH, '<path:csvfile>'),
-            rad_batch_csv_download_view,
-            name='serve_private_file',
-        ),
+        path('api/v1/', include(get_private_store_urls())),
+        path('radius/social-login/', include(get_social_urls(social_views))),
+        path('radius/saml2/', include(get_saml_urls(saml_views))),
     ]
-    if app_settings.SOCIAL_LOGIN_ENABLED:
-        urls.append(
-            path(
-                'radius/social-login/<slug:slug>/',
-                social_views.redirect_cp,
-                name='redirect_cp',
-            )
-        )
-    if app_settings.SAML_LOGIN_ENABLED:
-        from .saml.urls import get_saml_urls
-
-        urls.append(path('radius/saml2/', include(get_saml_urls(saml_views))),)
-    return urls
 
 
-app_name = 'radius'
-urlpatterns = get_urls()
+# Cannot used 'app_name' here because when the radius
+# urls are not registered, yet we want to get url reverse
+# using urlconf we still want radius to be our namespace.
+namespace = 'radius'
+urlpatterns = [path('', include((get_urls(), namespace), namespace=namespace))]
```

Comment:
So, here is the solution, I've come up with.
I don't use `app_name` instead, I explicitly mention the namespace.
Hence, when radius urls are not registered, I'll still get the `radius:` namespace when using the urls with the help of `urlconf=openwisp.urls`.

I need the radius namespace because some `reverse` requests are trying to look for urls in that namespace which gets registered when the registered with the help of `app_name`!